include/fastdds/rtps/messages/CDRMessage.hpp

```diff
@@ -265,11 +264,13 @@ inline SequenceNumberSet_t CDRMessage::readSequenceNumberSet(
 
     if (valid)
     {
+        SequenceNumberSet_t sns(seqNum, numBits);
         sns.base(seqNum);
```

Comment:
Is it necessary this line now? I think the constructor of BitmapRage makes this work now.

Reply:
You are right! Fixed!